### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by jaimevillegas**
Hi There!

I have made the following changes to the code: :fire:

- Add `Website.js` with the header of the page. :white_check_mark:
- Add `Home.js` to show index information. :memo:
- Add React Router to `index.js` to configure all the routers. :twisted_rightwards_arrows:
- Add styling according to the wireframe using Import on the index.js :art:

Thank you for taking the time to review my code!

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -11,6 +11,55 @@ body {
...
+.home-wrapper {
+  padding: 80px;
+}
+
+.calculator-wrapper {
+  width: 80%;
+  margin: auto;
+  border: 2px solid purple;
+  box-shadow: 2px 2px 8px purple;
+}
```
Comment (Lizdev-05) [inline src/index.css:61]: Great job so far 👍! Kindly give your calculator some margin bottom so as to match what we have on the wireframe... 🥺 

image

File: src/components/Website.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export default function Website() {
+  return (
+    <>
+      <header className="header-website">
+        <h2>Math Magicians</h2>
+        <nav>
+          <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="calculator">Calculator</a></li>
+            <li><a href="quote">Quote</a></li>
```
Comment (Lizdev-05) [inline src/components/Website.js:13]: Kindly check the quote URL, it keeps on saying "something went wrong" 😞 

image

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @jaimevillegas, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
-N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 1 above)
Comment (jaimevillegas) [reply src/index.css:61]: Hello! Thank you for the review! Task done!

image

File: src/components/Website.js (hunk 2 above)
Comment (jaimevillegas) [reply src/components/Website.js:13]: Hmmm, I think it is a server problem. It works for me 10/10 times:

image

image

image

image

image

**Review by Munsa1 — APPROVED** [review]
Hi @jaimevillegas, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made requested changes :heavy_check_mark: 

## Optional suggestions
- [ ] It would be nice if you could add the text `Lets do some math` so as to match the original design

Your design
image


Given frame 
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**